src/extension.ts

```diff
@@ -14,6 +14,7 @@ export function activate(context: vscode.ExtensionContext) {
     const regexRegex = /(^|\s|[()={},:?;])(\/((?:\\\/|\[[^\]]*\]|[^/])+)\/([gimuy]*))(\s|[()={},:?;]|$)/g;
     const phpRegexRegex = /(^|\s|[()={},:?;])['|"](\/((?:\\\/|\[[^\]]*\]|[^/])+)\/([gimuy]*))['|"](\s|[()={},:?;]|$)/g;
     const haxeRegexRegex = /(^|\s|[()={},:?;])(~\/((?:\\\/|\[[^\]]*\]|[^/])+)\/([gimsu]*))(\s|[.()={},:?;]|$)/g;
+    const pyRegexRegex = /(^|\s|[()={},:?;])['|"](((?:\\\/|\[[^\]]*\]|[^/])+)([gimuy]*))['|"](\s|[()={},:?;]|$)/g;
```

Comment:
I realize this shows the code lens above all strings. That seems a bit too much. Is there a way to reduce this to only match actual regular expressions? E.g., by only matching on strings as parameters to `re.search` and others?

Reply:
Can you provide a little more detail around what you're seeing and potential drawbacks?  This should be providing the same match/highlighting as the other languages.

Reply:
It seems to match strings that are not used as regular expressions. E.g.:
![image](https://user-images.githubusercontent.com/9205389/199180008-fd122855-8d03-4d52-832b-cdf497b40dbf.png)

Since strings are very common this would add a 'Test Regex...' link in many places where it doesn't actually makes sense. (Maybe you disabled code lenses and don't see that.)

Reply:
Would it be acceptable to change the regex to only match "raw" strings (e.g `r"\d{4}-\d{2}"`)? Sure it's not a perfect solution, as we would miss some simple regexes and also have a bunch of false positives, but if a caveat is added in the doc, it could work.